recipe/meta.yaml

```diff
@@ -42,11 +42,11 @@ requirements:
 
 test:
   requires:
-    - nose
+    - pytest
   imports:
     - sklearn
   commands:
-    - nosetests sklearn --exe
+    - pytest --pyargs sklearn
     - conda inspect linkages -p $PREFIX scikit-learn  # [not win]
     - conda inspect objects -p $PREFIX scikit-learn  # [osx]
```

Comment:
We can drop the `conda inspect` lines as `conda-build` does this for us now. Plus those commands will soon be deprecated and removed.